pkg/kp/pcstore/consul_store.go

```diff
@@ -154,6 +157,76 @@ func (s *consulStore) Delete(id fields.ID) error {
 	return s.applicator.RemoveAllLabels(labels.PC, id.String())
 }
 
+// performs a safe (ie check-and-set) mutation of the pc with the given id,
+// using the given function
+// if the mutator returns an error, it will be propagated out
+// if the returned PC has id="", then it will be deleted
+func (s *consulStore) mutatePc(
```

Comment:
In Go, functions with lowercase first letters aren't exposed outside the package (in this case, the package is `pcstore`. We'll want to be calling this from our internal repo, so you want to name this `MutatePC`